cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/runtime/config/RuntimeConfiguration.java

```diff
@@ -61,10 +61,11 @@
 
     public static boolean isLocalEnvironment(final Environment environment) {
         for (final String profile : environment.getActiveProfiles()) {
-            if (profile.equalsIgnoreCase(LOCAL_PROFILE)) {
+            if (profile.equalsIgnoreCase(LOCAL_PROFILE) || profile.equalsIgnoreCase(TEST_PROFILE)
```

Comment:
It's also used to check if we need to find out the EC2 instance name.

Perhaps we need a "isDeployedEnvironment()" method instead?